2025-03-31 21:09:01 INFO log.py:21: Yasb v1.6.8
2025-03-31 21:09:02 DEBUG win32.py:55: Looking up time zone info from registry
2025-03-31 21:09:04 DEBUG __init__.py:149: CoInitializeEx(None, 2)
2025-03-31 21:09:04 INFO weather.py:369: Fetched new weather data at 2025-03-31 21:09:04.449862
2025-03-31 21:09:04 ERROR weather.py:472: Error fetching weather data: HTTP Error 403: Forbidden
2025-03-31 21:09:05 INFO app_bar.py:103: Bar Created on Screen: Odyssey G95SC [Bar Height: 30px, DPI Scale: 1.0, Scale Screen: True, Screen Geometry: X: 0, Y: 0, Screen Width: 5120, Screen Height: 1440]
2025-03-31 21:09:05 ERROR github.py:410: HTTP Error occurred: 401 - {"message":"Bad credentials","documentation_url":"https://docs.github.com/rest","status":"401"}
2025-03-31 21:09:05 INFO weather.py:369: Fetched new weather data at 2025-03-31 21:09:05.923218
2025-03-31 21:09:05 INFO app_bar.py:103: Bar Created on Screen: L1 [Bar Height: 30px, DPI Scale: 1.0, Scale Screen: True, Screen Geometry: X: 621, Y: 1440, Screen Width: 1920, Screen Height: 860]
2025-03-31 21:09:06 ERROR weather.py:472: Error fetching weather data: HTTP Error 403: Forbidden
2025-03-31 21:09:06 INFO event_listener.py:52: Created named pipe yasb-4a2e4d78-0e9e-11f0-8a64-fcb0de6eb67e
2025-03-31 21:09:06 ERROR github.py:410: HTTP Error occurred: 401 - {"message":"Bad credentials","documentation_url":"https://docs.github.com/rest","status":"401"}
2025-03-31 21:09:06 INFO event_listener.py:106: Waiting for Komorebi to subscribe to named pipe yasb-4a2e4d78-0e9e-11f0-8a64-fcb0de6eb67e
2025-03-31 21:09:06 INFO cli_client.py:39: YASB CLI Pipe server started
2025-03-31 21:09:06 INFO watcher.py:39: Created file watcher for path C:\Users\eric\.config\yasb
2025-03-31 21:09:06 DEBUG _windows.py:213: Starting (proactor: <_IocpProactor overlapped#=0 result#=0>)...
2025-03-31 21:09:06 DEBUG _windows.py:192: Thread started
2025-03-31 21:09:06 INFO event_listener.py:118: Komorebi connected to named pipe: yasb-4a2e4d78-0e9e-11f0-8a64-fcb0de6eb67e
2025-03-31 21:12:25 INFO log.py:21: Yasb v1.6.8
2025-03-31 21:12:25 DEBUG win32.py:55: Looking up time zone info from registry
2025-03-31 21:12:26 DEBUG __init__.py:149: CoInitializeEx(None, 2)
2025-03-31 21:12:26 INFO weather.py:369: Fetched new weather data at 2025-03-31 21:12:26.135300
2025-03-31 21:12:26 ERROR weather.py:472: Error fetching weather data: HTTP Error 403: Forbidden
2025-03-31 21:12:26 INFO app_bar.py:103: Bar Created on Screen: Odyssey G95SC [Bar Height: 30px, DPI Scale: 1.0, Scale Screen: True, Screen Geometry: X: 0, Y: 0, Screen Width: 5120, Screen Height: 1440]
2025-03-31 21:12:26 ERROR github.py:410: HTTP Error occurred: 401 - {"message":"Bad credentials","documentation_url":"https://docs.github.com/rest","status":"401"}
2025-03-31 21:12:27 INFO weather.py:369: Fetched new weather data at 2025-03-31 21:12:27.221834
2025-03-31 21:12:27 INFO app_bar.py:103: Bar Created on Screen: L1 [Bar Height: 30px, DPI Scale: 1.0, Scale Screen: True, Screen Geometry: X: 621, Y: 1440, Screen Width: 1920, Screen Height: 860]
2025-03-31 21:12:27 ERROR weather.py:472: Error fetching weather data: HTTP Error 403: Forbidden
2025-03-31 21:12:27 ERROR github.py:410: HTTP Error occurred: 401 - {"message":"Bad credentials","documentation_url":"https://docs.github.com/rest","status":"401"}
2025-03-31 21:12:27 INFO event_listener.py:52: Created named pipe yasb-c23129a6-0e9e-11f0-9897-fcb0de6eb67e
2025-03-31 21:12:27 INFO event_listener.py:106: Waiting for Komorebi to subscribe to named pipe yasb-c23129a6-0e9e-11f0-9897-fcb0de6eb67e
2025-03-31 21:12:27 INFO cli_client.py:39: YASB CLI Pipe server started
2025-03-31 21:12:27 INFO watcher.py:39: Created file watcher for path C:\Users\eric\.config\yasb
2025-03-31 21:12:27 DEBUG _windows.py:213: Starting (proactor: <_IocpProactor overlapped#=0 result#=0>)...
2025-03-31 21:12:27 DEBUG _windows.py:192: Thread started
2025-03-31 21:12:27 INFO event_listener.py:118: Komorebi connected to named pipe: yasb-c23129a6-0e9e-11f0-9897-fcb0de6eb67e
2025-03-31 21:17:26 ERROR github.py:410: HTTP Error occurred: 401 - {"message":"Bad credentials","documentation_url":"https://docs.github.com/rest","status":"401"}
2025-03-31 21:17:27 ERROR github.py:410: HTTP Error occurred: 401 - {"message":"Bad credentials","documentation_url":"https://docs.github.com/rest","status":"401"}
2025-03-31 21:22:26 INFO weather.py:369: Fetched new weather data at 2025-03-31 21:22:26.136132
2025-03-31 21:22:26 ERROR weather.py:472: Error fetching weather data: HTTP Error 403: Forbidden
2025-03-31 21:22:26 ERROR github.py:410: HTTP Error occurred: 401 - {"message":"Bad credentials","documentation_url":"https://docs.github.com/rest","status":"401"}
2025-03-31 21:22:27 INFO weather.py:369: Fetched new weather data at 2025-03-31 21:22:27.218492
2025-03-31 21:22:27 ERROR weather.py:472: Error fetching weather data: HTTP Error 403: Forbidden
2025-03-31 21:22:27 ERROR github.py:410: HTTP Error occurred: 401 - {"message":"Bad credentials","documentation_url":"https://docs.github.com/rest","status":"401"}
2025-03-31 21:27:26 ERROR github.py:410: HTTP Error occurred: 401 - {"message":"Bad credentials","documentation_url":"https://docs.github.com/rest","status":"401"}
2025-03-31 21:27:27 ERROR github.py:410: HTTP Error occurred: 401 - {"message":"Bad credentials","documentation_url":"https://docs.github.com/rest","status":"401"}
